App.js

```diff
@@ -1,10 +1,33 @@
 import { StatusBar } from "expo-status-bar";
-import { StyleSheet, Text, View } from "react-native";
+import { Text, View, StyleSheet, TextInput } from "react-native";
+import { useEffect, useState } from "react";
+import { io } from "socket.io-client";
+import { SERVER_PORT } from "@env";
 
 export default function App() {
+  const [message, setMessage] = useState("");
+
+  useEffect(() => {
+    const socket = io(`${SERVER_PORT}`);
+    socket.emit("user-send", message);
+    console.log("useEffect ~ message", message);
+    socket.on("broadcast", (data) => {
+      console.log("socket.on ~ data", data);
+    });
```

Comment:
이 부분은 사실 제가 작성한 부분이긴 해서 제가 고쳐야할 점이긴 합니다.
Socket들이 서로 잘 통신되는지 시각적으로 확인하기 위해 console.log를 통해서 테스트를 했었는데요.
테스트가 끝난 이후에는 사실 필요가 없는 코드이니 지워야할것같습니다.
```suggestion
    const socket = io(`${SERVER_PORT}`);
    
    socket.emit("user-send", message);
    socket.on("broadcast");
```